runtime_config_linux.go

```diff
@@ -39,8 +39,10 @@ type LinuxRuntime struct {
 	SelinuxProcessLabel string `json:"selinuxProcessLabel"`
 	// Seccomp specifies the seccomp security settings for the container.
 	Seccomp Seccomp `json:"seccomp"`
-	// RootfsPropagation is the rootfs mount propagation mode for the container
+	// RootfsPropagation is the rootfs mount propagation mode for the container.
 	RootfsPropagation string `json:"rootfsPropagation,omitempty"`
+	// disableNewPrivileges controls whether additional privileges could be gained by processes in the container.
+	DisableNewPrivileges `json:"disableNewPrivileges,omitempty"`
```

Comment:
My suggested generic wording for null/missing is [here][1].

[1]: https://github.com/opencontainers/specs/pull/285#issuecomment-167823651